features/step_definitions/job.rb

```diff
@@ -4,3 +4,17 @@
 
   raise "job #{job.name} never completed" unless @result[:success]
 end
+
+Given /^a job is present with labels:$/ do |table|
```

Comment:
can we use existing step instead of writing a new one?
```
/^(I|admin) waits? for the #{QUOTED} (\w+) to appear(?: in the#{OPT_QUOTED} project)?(?: up to (\d+) seconds)?$/
```